index.ts

```diff
@@ -10,3 +10,5 @@ import './view/code/index.js'
 import './view/checkbox/index.js'
 import './view/settings/index.js'
 import './view/benchmark/index.js'
+import './view/plot-3d/index.js'
```

Comment:
You added 400 KB JS by this line.

We should add load `three.js` only if user asked to show 3D space.